src/rgw/rgw_common.h

```diff
@@ -323,6 +323,10 @@ static inline const char* to_mime_type(const RGWFormat f)
 
 typedef void *RGWAccessHandle;
 
+std::string flag_summary(std::vector<std::pair<int,std::string>> flag_list,
+			 int64_t flags,
+			 bool include_missing = false);
+
```

Comment:
`RGWBucketInfo::dump()` is the only caller, so i don't think this needs to be exposed to the rest of rgw in rgw_common.h. the `include_missing` functionality is unused - do you have plans to use this stuff elsewhere?